ZAPDTR/ZAPD/ZResource.cpp

```diff
@@ -304,8 +305,13 @@ void ZResource::GetSourceOutputCode([[maybe_unused]] const std::string& prefix)
 	}
 }
 
-std::string ZResource::GetSourceOutputHeader([[maybe_unused]] const std::string& prefix)
+std::string ZResource::GetSourceOutputHeader([[maybe_unused]] const std::string& prefix, bool first)
 {
+	static std::set<std::string> totalset;
```

Comment:
This is used to remember which resources have already been processed right? Let's use a more descriptive name.

Reply:
Also now this out of my c++ expertise but if this is static shouldn't we declare it outside of this function. How does declaring a static variable work when it happens inside a function that can be called multiple times? Or is it still statically scoped only to this function? That would be cool.

Reply:
changed it to take a set pointer which makes this clearer hopefully